Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -189,11 +188,16 @@ class FieldValue : public util::Comparable<FieldValue> {
   static FieldValue FromArray(std::vector<FieldValue>&& value);
   static FieldValue FromMap(const Map& value);
   static FieldValue FromMap(Map&& value);
+  static FieldValue FromMap(
```

Comment:
Hmm, `SortedMap`'s constructor taking an `initializer_list` is not `explicit`, so it seems like `FromMap({"foo", {}})` should have worked already. Was it not the case?